src/model/OWL.pl

```diff
@@ -319,6 +334,15 @@
 is_complement_of(ComplementClass,not(Class)) ?>
   triple(ComplementClass,owl:complementOf,Class).
 
+%% is_all_disjoint_classes(?AllDisjointClasses) is nondet.
+%
+% True for OWL2 AllDisjointClasses
+%
+% @param Entity An entity IRI.
+%
+is_all_disjoint_classes(Entity) ?+>
```

Comment:
maybe also export the predicate in the module definition? Probably would be appropriate as this is part of OWL spec